.vscode/launch.json

```diff
@@ -4,6 +4,20 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
+    {
+      "name": "Launch via NPM",
+      "request": "launch",
+      "runtimeArgs": [
+        "run-script",
+        "docs-collect"
+      ],
+      "runtimeExecutable": "npm",
+      "skipFiles": [
+        "<node_internals>/**"
+      ],
+      "type": "pwa-node"
+    },
```

Comment:
there's a new auto attach feature in vscode debugger, there's no need to specify a launch.json config if it's just for temporary use

Reply:
https://github.com/microsoft/vscode-js-debug#debug-nodejs-processes-in-the-terminal